File: app/src/main/java/com/capstone/project_niyakneyak/ui/main/listener/OnToggleAlarmListener.java
package com.capstone.project_niyakneyak.ui.main.listener;

import com.capstone.project_niyakneyak.data.alarm_model.Alarm;

public interface OnToggleAlarmListener {
    void onToggle(Alarm alarm);
    void onDelete(Alarm alarm);
    void onItemClick(Alarm alarm);
}

File: app/src/main/java/com/capstone/project_niyakneyak/ui/main/listener/OnToggleAlarmListener.kt
package com.capstone.project_niyakneyak.ui.main.listener

import com.capstone.project_niyakneyak.data.alarm_model.Alarm

interface OnToggleAlarmListener {
    fun onToggle(alarm: Alarm)
    fun onDelete(alarm: Alarm)
    fun onItemClick(alarm: Alarm)
}